setup.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# pylint: disable=C0111,W6005,W6100
```

Comment:
You might want to pick up this change: https://github.com/edx/edx-django-utils/blob/master/setup.py#L3-L5